maven-example/cloudbuild.yaml

```diff
@@ -20,6 +20,6 @@ steps:
     entrypoint: mvn
     args: ["package", "-Dmaven.test.skip=true"]
   - name: gcr.io/cloud-builders/docker
-    args: ["build", "-t", "gcr.io/$PROJECT_ID/hellospring", "--build-arg=JAR_FILE=target/hello-spring-1.0.0.jar", "."]
-images: ["gcr.io/$PROJECT_ID/hellospring"]
-# [END cloudbuild_maven]
\ No newline at end of file
+    args: ["build", "-t", "my_host_location-docker.pkg.dev/$PROJECT_ID/my_repository/hellospring", "--build-arg=JAR_FILE=target/hello-spring-1.0.0.jar", "."]
```

Comment:
This change is failing because these configurations are tested, and "my_host_location" isn't a real location.